scripts.js

```diff
@@ -36,10 +36,38 @@ $(".save").on("click", function() {
 })
 
 function applyInput(newIdea) {
-    $('.ideabody').prepend(`<div id=${newIdea.id} class = ${newIdea.class}><span class = "delete"><h2>${newIdea.title}<button class = "deleteArrow"></button></h2></span><p class = "body-content">${newIdea.body}</p><p class = "quality"><button class = "upvote"></button><button class = "downvote"></button> quality: ${newIdea.quality}</p><hr></div>`)
+    $('.ideabody').prepend(`<div id=${newIdea.id} class = ${newIdea.class}><span class = "delete"><h2 class="editable-title" contenteditable="true">${newIdea.title}<button class = "deleteArrow"></button></h2></span><p contenteditable="true" class = "editable-body">${newIdea.body}</p><p class = "quality"><button class = "upvote"></button><button class = "downvote"></button> quality: ${newIdea.quality}</p><hr></div>`)
 }
 
 
+    $(".ideabody").on("keypress", ".editable-title", function (e) {
+      if (e.which == 13) {
+        e.preventDefault();
+        $(this).blur();
+        console.log('blur')
+      }
+    });
+
```

Comment:
@AKewlBreeze Yes, on the keypress event listener we're preventing the enter button's default and selecting that content and calling blur, so that resolves that issue.